docs/wiki/99_Developers/LORIS_Glossary.md

```diff
@@ -0,0 +1,107 @@
+**Administration:** in an instrument form, can be marked “None”, “Partial”, or “All”. If an instrument was not administered, it is marked as “None.” This will remove the instrument from the database statistics calculation. If an instrument was partially administered (i.e. candidate did not answer some questions, parent did not provide some answers, tester did not ask some of the questions), it is marked as “Partial”. This field has to be set for Data Entry to be marked as “Complete”.
```

Comment:
Can you add an introductory line like : 
`Below is a list of terms used mostly in the context of the LORIS user-facing front-end used.  For definition of Study variables used in configuring LORIS also see ` [ then add  a link to the Study Variables part where project, subproject/cohort etc are defined. ]